test/Sets/SimpleSparsePolynomialZonotope.jl

```diff
@@ -32,4 +32,25 @@ for N in [Float64, Float32, Rational{Int}]
     @test genmat(CPS) == [1 2 0 0;2 2 0 0;0 0 1 2;0 0 2 2.]
     @test expmat(CPS) == [1 4 0 0;1 2 0 0;0 0 1 4;0 0 1 2]
 
+    _c = N[2.0, 0.0]
```

Comment:
It is okay now, but typically we do not use underscores for variables. Since we have the outer loops in the test files, there are usually no scoping problems.